+sw_tests/+unit_tests/unittest_spinw_addaniso.m

```diff
@@ -0,0 +1,83 @@
+classdef unittest_spinw_addaniso < matlab.mock.TestCase
+
+    properties
+        swobj = [];
+        make_single_ion_struct_with_aniso = @(aniso) struct(...
```

Comment:
Any reason why this isn't a method? I feel like it might be more readable...

You can have a `methods` block without the `(Test)` marker